lib/pages/share/share_view.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:fluffychat/pages/forward/recent_chat_list.dart';
+import 'package:fluffychat/pages/forward/recent_chat_title.dart';
+import 'package:fluffychat/pages/share/share.dart';
+import 'package:fluffychat/widgets/matrix.dart';
+import 'package:fluffychat/widgets/twake_components/twake_fab.dart';
+import 'package:fluffychat/widgets/twake_components/twake_icon_button.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:go_router/go_router.dart';
+import 'package:matrix/matrix.dart';
+
+class ShareView extends StatelessWidget {
+  const ShareView(this.controller, {super.key});
+
+  final ShareController controller;
```

Comment:
Remember that don't use `controller` in `StatelessWidget`

Reply:
no, This's big controller